tests/snapshots/framework-detection.test.js.md

```diff
@@ -158,4 +168,4 @@ Generated by [AVA](https://avajs.dev).
     > start␊
     > react-scripts start␊
     ␊
-    ◈ "npm start" exited with code *. Shutting down Netlify Dev server`
+    ◈ Failed launching framework server. Please verify 'npm' exists`
```

Comment:
This doesn't look good. Looking into it

Reply:
Fixed in https://github.com/netlify/cli/pull/2857/commits/f41e5b1946e2bc6d317aefb85b81b0f3d7f166c5